features/frameworks/mbed-coap/source/sn_coap_parser.c

```diff
@@ -310,16 +310,30 @@ static int8_t sn_coap_parser_options_parse(struct coap_s *handle, uint8_t **pack
             option_number = *(*packet_data_pptr + 1) + 13;
             (*packet_data_pptr)++;
         } else if (option_number == 14) {
-            option_number = *(*packet_data_pptr + 2);
-            option_number += (*(*packet_data_pptr + 1) << 8) + 269;
-            (*packet_data_pptr) += 2;
+            if ( message_left>= 2){
```

Comment:
can you fix , most of changed conditions have extra space `if (message_left` should be, same below on line 345

Reply:
Fixed extra spaces